ldapadmin/src/deb/resources/etc/georchestra/ldapadmin/css/main.css

```diff
@@ -0,0 +1,123 @@
+#container {
```

Comment:
That is what we already did for GEOR_custom.js anyway (more relevant in this case though)